src/ansi-c/ansi_c_language.h

```diff
@@ -17,10 +17,16 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <langapi/language.h>
 
 #include "ansi_c_parse_tree.h"
+#include "c_object_factory_parameters.h"
 
 class ansi_c_languaget:public languaget
 {
 public:
+  void set_language_options(const optionst &options) override
```

Comment:
Added but never invoked?

Reply:
This would e.g. be called by `initialize_goto_model()` where `languaget &` might reference `ansi_c_languaget`.